src/snowflake/cli/plugins/spcs/compute_pool/commands.py

```diff
@@ -114,3 +121,99 @@ def resume(name: str = ComputePoolNameArgument, **options) -> CommandResult:
     Resumes the compute pool from SUSPENDED state.
     """
     return SingleQueryResult(ComputePoolManager().resume(name))
+
+
+@app.command("set")
+@with_output
+@global_options_with_connection
+def set_property(
+    name: str = ComputePoolNameArgument,
+    min_nodes: Optional[int] = typer.Option(
+        None,
+        "--min-nodes",
+        help="Minimum number of nodes for the compute pool.",
+        min=1,
+        show_default=False,
+    ),
+    max_nodes: Optional[int] = typer.Option(
+        None,
+        "--max-nodes",
+        help="Maximum number of nodes for the compute pool.",
+        min=1,
+        show_default=False,
+    ),
+    auto_resume: Optional[bool] = typer.Option(  # type: ignore
+        None,
+        "--auto-resume/--no-auto-resume",
+        help="The compute pool will automatically resume when a service or job is submitted to it.",
+        show_default=False,
+    ),
+    auto_suspend_secs: Optional[int] = typer.Option(
+        None,
+        "--auto-suspend-secs",
+        help="Number of seconds of inactivity after which you want Snowflake to automatically suspend the compute pool.",
+        min=1,
+        show_default=False,
+    ),
+    comment: Optional[str] = comment_option("compute pool"),
```

Comment:
`create` and `set` have common options, lets extract them like in src/snowflake/cli/api/commands/flags.py, but keep them in module.

Reply:
good point, there are some small differences between the options used in each (default, show_default), so I've created a generic function that creates a typer Option with some default arguments while also allowing you to override select arguments.